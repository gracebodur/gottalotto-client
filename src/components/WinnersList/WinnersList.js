import React, { Component } from 'react'

class WinnersList extends Component {
	render() {
		return (
			<div>
				<main>
					<h2>Most correct numbers</h2>
					<section>
						<div className='winnerList'>
							<div>
								<img src='https://via.placeholder.com/50' alt='users avatar' />
							</div>
							<h3 className='winnerName'>Grace Bodur</h3>
							<h3 className='winnerGuess' s>05, 08, 16, 18, 26, 109</h3>
							<p className='winnerMessage'>If you have a few dollars to spare, donate to this wonderful cause: https://comngood.org/home</p>
						</div>
						<div className='winnerList'>
							<div>
								<img src='https://via.placeholder.com/50' alt='users avatar' />
							</div>
							<h3 className='winnerName'>Malek Haj-Hussein</h3>
							<h3 className='winnerGuess'>06, 09, 16, 19, 29, 55</h3>
							<p className='winnerMessage'>Check out my social media pages! https://www.linkedin.com/in/malekadair/</p>
						</div>
						<div className='winnerList'>
							<div>
								<img src='https://via.placeholder.com/50' alt='users avatar' />
							</div>
							<h3 className='winnerName'>Gary</h3>
							<h3 className='winnerGuess'>07, 07, 07, 07, 07, 07</h3>
							<p className='winnerMessage'>A'jahn is cool.</p>
						</div>
					</section>
				</main>
			</div>
		)
	}
}

export default WinnersList