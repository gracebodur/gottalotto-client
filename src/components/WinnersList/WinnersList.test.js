import React from "react";
import toJson from "enzyme-to-json";
import WinnersList from "./WinnersList";

describe(`WinnersList component`, () => {
	const props = {
		id: "a",
		name: "test-class-name"
	};

	it("renders a WinnersList by default", () => {
		const wrapper = <WinnersList />;
		expect(toJson(wrapper)).toMatchSnapshot();
	});

	it("renders the WinnersList given props", () => {
		const wrapper = <WinnersList {...props} />;
		expect(toJson(wrapper)).toMatchSnapshot();
	});
});
