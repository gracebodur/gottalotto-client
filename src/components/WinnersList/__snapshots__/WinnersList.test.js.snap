// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WinnersList component renders a WinnersList by default 1`] = `undefined`;

exports[`WinnersList component renders the WinnersList given props 1`] = `undefined`;
